server/src/repository/round.ts

```diff
@@ -8,7 +8,7 @@ export class RoundRepository {
     this.repo = dataSource.getRepository(Round);
   }
 
-  async getRounds(limit: number, offset: number): Promise<[Round[], number]> {
+  async getPaginatedRounds(limit: number, offset: number): Promise<[Round[], number]> {
```

Comment:
This method accepts two separate pagination params, while the methods in the `Play` repository accept an object. Let's make those consistent and use the same signature `(pagination, filters)`